test/integration/commit_int_test.go

```diff
@@ -6,7 +6,6 @@ import (
 	"testing"
 
 	"github.com/ActiveState/cli/internal/constants"
-	"github.com/ActiveState/cli/internal/errs"
```

Comment:
Oops, this is still needed.

Reply:
You had one job Goland! 

Well actually a lot of jobs, and I probably just confused the hell out of it. But it's easier to blame Goland..